docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+
+```xml
+<!ELEMENT foreignSpaceReplacement ( #PCDATA ) >
+<!ATTLIST foreignSpaceReplacement xml:space preserve #REQUIRED >
+```
+
+* `xml:space` must be set to `'preserve'` so that actual spaces in the pattern are preserved. See [W3C XML White Space Handling](https://www.w3.org/TR/xml/#sec-white-space).
+* The `#PCDATA `is the character sequence used to replace spaces between fields for name data from a name locale that is different from the formatting locale, but are in the same script.
+
+### 2.5 <a name="2-5-initialpattern-element" href="#2-5-initialpattern-element">initialPattern Element</a>
+
+The `<initialPattern>` element is used to specify how to format initials of name parts.
+
+**_initial_** is a pattern used to display a single initial in the locale, while **_initialSequence_ **is a pattern used to “glue” together multiple initials for multiword fields, for example with the given name “Mary Beth” in English.
+
+#### 2.5.1 <a name="syntax" href="#syntax">Syntax</a>
+
+```xml
+<!ELEMENT initialPattern ( #PCDATA ) >
+<!ATTLIST initialPattern type ( initial | initialSequence) #REQUIRED >
+```
+
+The `type="initial"` is used to specify the pattern for how single initials are created, for example “Wolcott” => “W.” would have an entry of
+
+```xml
+<initialPattern type="initial">{0}.</initialPattern>
+```
+
+`type="initialSequence`” is used to specify how a series of initials should appear, for example “Wolcott Janus” => “W. J.”, with spaces between each initial, would have a specifier of
+
+```xml
+<initialPattern type="initialSequence">{0} {1}</initialPattern>
+```
+
+## 3 <a name="3-person-name-object" href="#3-person-name-object">Person Name Object</a>
+
+The information that is to be formatted logically consists of a data object containing a number of fields. This data object is a construct for the purpose of formatting, and doesn’t represent the source of the name data. That is, the original source may contain more information. The PersonName object is merely a logical ‘transport’ of information to formatting; it may physically consist of, for example, an API that fetches fields from a database.
+
+Note that an application might have more than one set of name data for a given person, such as data for both a legal name and a nickname or preferred name. Or the source data may contain two whole sets of name data for a person from an Eastern Slavic region, one in Cyrillic characters and one in Latin characters. Or it might contain phonetic data for a name (commonly used in Japan). The additional application-specific information in person’s names is out of scope for the CLDR Person Name formatting data. Thus a calling application may produce more than one name object to format depending on the purpose. 
+
+For illustration, the following is a sample PersonName object.
+
+```
+person_name ⇒ 
```

Comment:
We probably want to make this a table. (just a nice-to-have)

Reply:
Yes, converted to table, done.